lib/trento/activity_logging/parser/metadata_enricher.ex

```diff
@@ -0,0 +1,125 @@
+defmodule Trento.ActivityLog.Logger.Parser.MetadataEnricher do
+  @moduledoc """
+  Metadata enricher enriches metadata extracted by activity parser.
+  """
+
+  alias Trento.ActivityLog.ActivityCatalog
+
+  alias Trento.Clusters.Projections.ClusterReadModel
+  alias Trento.Databases.Projections.DatabaseReadModel
+  alias Trento.Hosts.Projections.HostReadModel
+  alias Trento.SapSystems.Projections.SapSystemReadModel
```

Comment:
These are deep dependencies, not top level contexts. What do we lose if we do not assert on the type of the return (e.g. `Hosts.by_id()`) ?

Reply:
We wouldn't be losing much. Here we go a56f2fadf47014161506788a4defb2e62c9a47cc